samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/ChatHistoryController.cs

```diff
@@ -140,7 +161,7 @@ public async Task<IActionResult> GetAllChatSessionsAsync(string userId)
     {
         // TODO: the code mixes strings and Guid without being explicit about the serialization format
         var chatMessages = await this._chatMessageRepository.FindByChatIdAsync(chatId.ToString());
-        if (chatMessages == null)
+        if (!chatMessages.Any())
         {
             return this.NotFound($"No messages found for chat id '{chatId}'.");
```

Comment:
Should we just return an empty OK here, or do we consider a chat session without messages a failure?

Reply:
New chats should always have at least the initial bot message

Reply:
Making it explicit that a chat should have at least one message.